src/test/java/com/gymrats/GymRatsLogbookBackendApplicationTests.java

```diff
@@ -1,13 +1,26 @@
 package com.gymrats;
 
+import com.gymrats.model.BmiNotLoggedRequest;
+import com.gymrats.model.BmiNotLoggedResponse;
+import com.gymrats.model.GenderEnum;
+import com.gymrats.service.CalculatorService;
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
 
 @SpringBootTest
 class GymRatsLogbookBackendApplicationTests {
 
+    CalculatorService service = new CalculatorService();
+
     @Test
     void contextLoads() {
     }
 
+    @Test
+    void shouldReturnBmi() {
+        BmiNotLoggedRequest dataToCalculateBmi = new BmiNotLoggedRequest(GenderEnum.WOMAN, 60.0F, 170.0F);
```

Comment:
wiem że to tutaj wczesniej było, ale wyjmowałaś z target i komentarzy znikły,
//given
//when
//then